cmd/triggerrun/testdata/trigger.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: Trigger
+metadata:
+  name: trigger-run
+  namespace: ""
```

Comment:
As @dibyom mentioned here https://github.com/tektoncd/triggers/pull/695#discussion_r465795304 you no need to specify `namespace` because the default value of string is `""` so it will be`""` even if you don't mention explicitly 
So you can remove it as you have handled check in the code